### PR TITLE
remove one indirection for HttpServiceHandlerResponse

### PR DESCRIPTION
**Pull request by fakeshadow**
## PR Type

Refactor

## PR Checklist
Check your PR fulfills the following:

- [x] Format code with the latest stable rustfmt

## Overview

This indirect is not necessary.

### CODE REVIEW

File: actix-http/src/cloneable.rs [hunk 1]
```diff
@@ -10,7 +10,7 @@ use actix_service::Service;
 /// # Panics
 /// CloneableService might panic with some creative use of thread local storage.
 /// See https://github.com/actix/actix-web/issues/1295 for example
-pub(crate) struct CloneableService<T: Service>(Rc<RefCell<T>>);
+pub struct CloneableService<T: Service>(Rc<RefCell<T>>);
```
Comment (robjtede) [inline actix-http/src/cloneable.rs:13]: I'm of the opinion that it's easier to grok the scope of these when they're pub(crate) vs pub without a lib re-export.

Comment (fakeshadow) [reply actix-http/src/cloneable.rs:13]: It has to be public right? As HttpServiceHandlerResponse is now a public enum

Comment (robjtede) [reply actix-http/src/cloneable.rs:13]: aite, can we hide it from docs then. The DataFactory trait too, I don't want to assume semver responsibility for either of these things.

**Comment by fakeshadow** [comment]
Closing this for now. Looking back this is totally not a meaningful change.

**Pull request closed without merging**